sw/device/lib/dif/meson.build

```diff
@@ -476,6 +476,7 @@ test('dif_kmac_unittest', executable(
     sources: [
       'dif_kmac_unittest.cc',
       'autogen/dif_kmac_autogen_unittest.cc',
+      meson.source_root() / 'sw/device/lib/base/memory.c',
```

Comment:
I think this belongs in the other commit (the unittest change commit). You are not using it in the unittest (or drop it all-together and keep using `memcpy`, it's up to you).

Reply:
You do need `sw_lib_mem` dependency for the `sw_lib_dif_kmac` target.

Reply:
If I put it in the unittest commit, than the first commit won't compile.

Reply:
That is probably because of:
> You do need sw_lib_mem dependency for the sw_lib_dif_kmac target.

Reply:
Not really, there was already an unittest building for kmac, as I added the `read_32` function to the `dif_kmac.c` in the first commit the unittest needs to link the `memory.c`.

Reply:
My bad, you are right, you need this line here.